java.jdbc-ballerina/build.gradle

```diff
@@ -290,7 +299,7 @@ publishing {
     repositories {
         maven {
             name = "GitHubPackages"
-            url = uri("https://maven.pkg.github.com/ballerina-platform/module-ballerina-java.jdbc")
+            url = uri("https://maven.pkg.github.com/ballerina-platform/module-${packageOrg}-${packageName}")
```

Comment:
we can't use `packageOrg` here, because it refers to `ballerinax`

Reply:
Will do the change!